app/players/elves/druid.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        nick = self.nickname
+        level = self._favourite_spell
```

Comment:
no need to create variables for attributes, just break the line using brackets here